src/waveform/renderers/waveformrendermark.cpp

```diff
@@ -185,7 +185,11 @@ void WaveformRenderMark::generateMarkImage(WaveformMarkPointer pMark) {
     }
 
     QFont font; // Uses the application default, if not set per skin
```

Comment:
Please copy the description from the PR to preserve this information in the code.

Reply:
okay, done.